src/shared/locales/all.js

```diff
@@ -16,6 +18,12 @@ const resources = {
       ...appLocaleEnUS
     }
   },
+  'pr-BT': {
+    translation: {
+      ...eleLocalePtBr,
```

Comment:
Please keep the variable naming case uniform.
`eleLocalePtBr` => `eleLocalePtBR`

Reply:
Sorry. Fixed!